CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - added the ability to move the look camera while targeting an enemy in combat (#1187)
 - added the ability to skip fade-out in stats screens
 - added support for animated room sprites in custom levels and an option to animate plant sprites in The Cistern and Tomb of Tihocan (#449)
+- added OSD messages for certain actions (#1220)
```

Comment:
Might be worth writing out `on screen display` or `console`. Tbh, I didn't even know what OSD was at first til I googled it lol.